frontend/src/components/IstioWizards/K8sRequestRouting.tsx

```diff
@@ -262,6 +276,56 @@ class K8sRequestRouting extends React.Component<Props, State> {
     }
   }
 
+  onAddFilter = () => {
+    this.setState(prevState => {
+      let newFilter: string;
+      if (this.state.filterType === REQ_MOD || this.state.filterType === RESP_MOD) {
+        if (this.state.headerOp !== REMOVE) {
+          newFilter = prevState.filterType + ' [' + prevState.headerName + '] ' + prevState.headerOp + ' ' + prevState.headerValue;
```

Comment:
```suggestion
          newFilter = `${prevState.filterType} [${prevState.headerName}] ${prevState.headerOp} ${prevState.headerValue}`;
```

I can't find anything wrong in your code and so I'm just making subjective suggestions :-)  Again, no change required, but I find it less error prone to build strings in this way.